launch/test/launch_multiple_counters.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+
+from launch.exit_handler import IgnoreExitHandler
+from launch.exit_handler import RestartExitHandler
+from launch.output_handler import FileOutput
+from launch.output_handler import ConsoleOutput
+
+
+def launch(launch_descriptor):
+    counter_file = os.path.join(os.path.dirname(__file__), 'counter.py')
+
+    ld = launch_descriptor
+    ld.add_process(
+        cmd=[sys.executable, '-u', counter_file, '--limit', '9', '--sleep', '0.5'],
+        name='foo',
+        output_handlers=[FileOutput(filename='/tmp/foo.log')],
```

Comment:
You should probably use the `tempfile` module, `/tmp` may not exist or that file may already exist or be in use.